src/sparseml/pytorch/utils/exporter.py

```diff
@@ -745,25 +745,15 @@ def _delete_trivial_onnx_adds(model: onnx.ModelProto):
             continue
 
 
-def _fix_batch_norm_names(model: onnx.ModelProto):
-    name_to_inits = {init.name: init for init in model.graph.initializer}
+def _remove_module_from_names(model: onnx.ModelProto):
+    onnx.checker.check_model(model)
+
+    for init in model.graph.initializer:
+        init.name = init.name.replace(".module", "")
     for node in model.graph.node:
-        if node.op_type != "BatchNormalization":
```

Comment:
would prefer to keep scope still just to BN ops, `module` is still a valid name

Reply:
and i guess only initializers that go into batch norms?

Reply:
Okay I changed the name `module` to something else that should be way more specific to this wrapper. Prefer to keep the simpler logic of this function because the previous version was harder to understand than it needs to be